cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -326,6 +326,8 @@ data:
 
   # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
   ROOK_OBC_WATCH_OPERATOR_NAMESPACE: "true"
+  # The concurrency count of lib bucket provisioner
+  # ROOK_LIB_BUCKET_PROVISIONER_THREADS: "5"
```

Comment:
I would move this value up to the `env` section for the rook-ceph operator container, rename it to be the same env var name handled by the lib, so "LIB_BUCKET_PROVISIONER_THREADS", and remove the rest of the code related to this. What is the reason for implementing it the way done in this PR?

Reply:
My initial patch was setting env variable from rook code, never thought of using env variables in the pod until @travisn mentioned

Reply:
I agree with @travisn (and @copejon wrote a similar comment earlier) that the env var should be set in the operator manifest. With that done I believe the rest of the changes in this